Modules/Daq/src/DaqTask.cxx

```diff
@@ -100,41 +100,42 @@ void DaqTask::startOfCycle()
   QcInfoLogger::GetInstance() << "startOfCycle" << AliceO2::InfoLogger::InfoLogger::endm;
 }
 
-void DaqTask::monitorDataBlock(DataSetReference dataSet)
+void DaqTask::monitorData(o2::framework::ProcessingContext& ctx)
 {
-  if (dataSet->empty()) {
-    cout << "Empty vector!" << endl;
-    return;
-  }
+  QcInfoLogger::GetInstance() << "initialize SkeletonTaskDPL" << AliceO2::InfoLogger::InfoLogger::endm;
+
+
+//what does it mean to have several inputs ? is it that we defined several in the config file ?
+// If I am connected to the readout can I ever receive several inputs ?
```

Comment:
Theoretically, one could also receive several inputs along with readout. However, they should be in the same timebase (`timesliceID`) , so DPL can match them. Then, they would be accesible by their binding:
```
auto a = ctx.inputs().get("readout");
auto b = ctx.inputs().get("something-after-readout");